draft-ietf-teep-protocol.md

```diff
@@ -584,14 +584,14 @@ requirements, whether these claims appear in Attestation Results, or in Evidence
 for the Verifier to use when generating Attestation Results of some form:
 
 | Requirement  | Claim | Reference |
-| Freshness proof | nonce | {{I-D.ietf-rats-eat}} section 4.1 |
+| Freshness proof | eat_nonce | {{I-D.ietf-rats-eat}} section 4.1 |
 | Device unique identifier | ueid | {{I-D.ietf-rats-eat}} section 4.2.1 |
 | Vendor of the device | oemid | {{I-D.ietf-rats-eat}} section 4.2.3 |
-| Class of the device | hardware-model | {{I-D.ietf-rats-eat}} section 4.2.4 |
-| TEE hardware type | hardware-version | {{I-D.ietf-rats-eat}} section 4.2.5 |
-| TEE hardware version | hardware-version | {{I-D.ietf-rats-eat}} section 4.2.5 |
-| TEE firmware type | manifests | {{I-D.ietf-rats-eat}} section 4.2.16 |
-| TEE firmware version | manifests | {{I-D.ietf-rats-eat}} section 4.2.16 |
+| Class of the device | hwmodel | {{I-D.ietf-rats-eat}} section 4.2.4 |
+| TEE hardware type | hwersion | {{I-D.ietf-rats-eat}} section 4.2.5 |
```

Comment:
EAT claims only support a "hw version", not "hw type" Is "hardware type" necessary in TEEP or EAT should add a support? For non-EAT claim, is "hardware type" necessary? It looks intuitive when we started the spec where the underlying hardware type is asked first and then its version.

Reply:
"hardware type" can be derived from oemid and hwmodel so is not needed in addition

Reply:
So are you going to remove it in the list? It currently is still listed and refers to the same "hwversion" as the "TEE hardware version"

Reply:
The architecture document has:

> TEE Identifying Information: The type of TEE that generated this attestation must be identified. This includes version identification information for hardware, firmware, and software version of the TEE, as applicable by the **TEE type**. TEE manufacturer information for the TEE is required in order to disambiguate the same **TEE type** created by different manufacturers and address considerations around manufacturer provisioning, keying and support for the TEE.

So I don't want to remove it entirely.   And after re-reading, I misspoke, sometimes you need the hwversion to know the TEE type, if you consider the TEE "version" as being part of the TEE type.  I was trying to avoid any changes to the architecture draft.